conf/ma/demo/baseline/cifar100_res18.yaml

```diff
@@ -7,7 +7,7 @@ train_epochs: 1
 
 data_set: "cifar100"
 data_path:  "/home/vmagent/app/data/dataset/cifar"
-num_workers: 4
+num_workers: 0
```

Comment:
is 0 correct？

Reply:
Yes, 0 is torch default value for num_worker.